### PR TITLE
feat(chrome): add SkipNav to examples

### PR DESCRIPTION
**Pull request by jzempel**
## Description

- upgrade to `react-components` v8.31.0
- upgrade to `svg-icons` v6.28.0
- add `SkipNav` component to Chrome examples
- add `SkipNav` functionality to the website "Titled" pages

## Checklist

- [x] :ok_hand: website updates are Garden Designer approved (add the designer as a reviewer)
- [ ] :black_nib: ~copy updates are approved (add the content strategist as a reviewer)~
- [ ] :link: ~considered opportunities for adding cross-reference URLs (grep for keywords)~
- [x] :wheelchair: analyzed via [axe](https://www.deque.com/axe/) and evaluated using VoiceOver
- [x] :memo: tested in Chrome, Firefox, Safari, Edge, and IE11

### CODE REVIEW

File: src/examples/theme-provider/ThemeProviderBranding.tsx [hunk 1]
```diff
@@ -8,11 +8,11 @@
 import React from 'react';
 import { Row, Col } from '@zendeskgarden/react-grid';
 import { Button } from '@zendeskgarden/react-buttons';
-import { DefaultTheme, ThemeProvider } from '@zendeskgarden/react-theming';
+import { IGardenTheme, ThemeProvider } from '@zendeskgarden/react-theming';
```
Comment (jzempel) [inline src/examples/theme-provider/ThemeProviderBranding.tsx:11]: @austingreendev please double-check that I got this usage right throughout :pray:

**Review by allisonacs — APPROVED** [review]
Looks good to me. I'll make a ticket to include skip nav info in some kind of pattern doc, whether that's nav or accessibility.

File: src/examples/theme-provider/ThemeProviderBranding.tsx (hunk 1 above)
Comment (austingreendev) [reply src/examples/theme-provider/ThemeProviderBranding.tsx:11]: With the changes already in `styled.d.ts` I don't believe we need to retrieve `IGardenTheme` here. You should be able to use the following instead.

```
import { DefaultTheme } from 'styled-components';   
```

Comment (jzempel) [reply src/examples/theme-provider/ThemeProviderBranding.tsx:11]: Wasn't sure whether to optimize for `website` or `codesandbox`. But this makes sense.

Comment (austingreendev) [reply src/examples/theme-provider/ThemeProviderBranding.tsx:11]: Within codesandbox the `DefaultTheme` value is `any` when it's not extended. So it should continue to work fine!

**Review by austingreendev — APPROVED** [review]

**Review by hzhu — APPROVED** [review]

**Pull request merged**